test/parallel/test-tls-delayed-attach.js

```diff
@@ -24,6 +24,8 @@ const common = require('../common');
 if (!common.hasCrypto)
   common.skip('missing crypto');
 
+// Checks tls delay
```

Comment:
Can we get a more descriptive comment here? This doesn't add any information that isn't in the file name.

Reply:
more description added